features/yield/FarmingCalculator.tsx

```diff
@@ -38,33 +54,34 @@ const FarmingCalculator = () => {
   );
 
   // Yield inputs:
+  const currentTokenPrice = getTokenPrice(tokenAddress);
```

Comment:
This was moved from lower down. FYI both `getTokenPrice` and `getPoolData` are wrappers around graphQL hooks and therefore cannot be called within other hooks like `useEffect`

Reply:
Just curious, why can't it be called within a `useEffect`?

Reply:
Calling hooks within hooks violates the rule of hooks (that was a mouthful): https://github.com/facebook/react/issues/16582#issuecomment-526807226

I had this same error